recipes/llvm-core/all/conandata.yml

```diff
@@ -10,18 +10,57 @@ sources:
     sha256: ce8508e318a01a63d4e8b3090ab2ded3c598a50258cc49e2625b9120d4c03ea5
 
 patches:
-  "11.1.0":
-    - base_path: "source"
-      patch_file: "patches/11x/11.1.0-cmake.patch"
-    - base_path: "source"
-      patch_file: "patches/11x/11.1.0-native.patch"
-  "12.0.0":
-    - base_path: "source"
-      patch_file: "patches/12x/12.0.0-cmake.patch"
-    - base_path: "source"
-      patch_file: "patches/12x/12.0.0-native.patch"
   "13.0.0":
-    - base_path: "source"
-      patch_file: "patches/13x/13.0.0-cmake.patch"
-    - base_path: "source"
-      patch_file: "patches/13x/13.0.0-native.patch"
+    - patch_file: patches/13x/0000-cmake-dependencies.patch
+      patch_description: fix references to third party libs to match conan variables and targets
+      patch_type: conan
+    - patch_file: patches/13x/0001-calculate-job-pools.patch
+      patch_description: calculate resource limits for compiling LLVM
+      patch_type: portability
+      patch_source: https://github.com/llvm/llvm-project/pull/65274
+    - patch_file: patches/13x/0002-missing-includes.patch
+      patch_description: fix missing header files
+      patch_type: portability
```

Comment:
Just to double-check, does this patch come from https://github.com/llvm/llvm-project-release-prs/pull/3?

Reply:
Yes - updated `patch_source`